docs/app/Examples/elements/Rail/Variations/RailSizeExample.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import { Image, Grid, Rail, Segment } from 'stardust'
+
+const RailSizeExample = () => (
+  <Grid className='three column'>
+    <Grid.Column className='centered'>
+      <Segment>
+        <Image src='http://semantic-ui.com/images/wireframe/paragraph.png' />
+
+        <Rail size='tiny' position='left'>
+          <Segment>Left Rail Content</Segment>
+        </Rail>
+
+        <Rail size='tiny' position='right'>
+          <Segment>Right Rail Content</Segment>
+        </Rail>
+      </Segment>
+    </Grid.Column>
+  </Grid>
+)
```

Comment:
Also, I've submitted https://github.com/Semantic-Org/Semantic-UI-Docs/issues/205 to have this fixed.  In doing so, I captured all sizes in one screenshot and noticed an oversight I made previously.  The Rail does in fact supports all sizes except `medium`, they were staggered and spread out in the SUI docs:

![image](https://cloud.githubusercontent.com/assets/5067638/16853365/eb4ed6ac-49c0-11e6-9c1e-c3ffb5fa5dff.png)